packages/slice-machine/helpers/storybook.js

```diff
@@ -0,0 +1,16 @@
+/** Called from user project */
+import glob from "glob";
```

Comment:
use require as this will be called as a common js module :)

Reply:
mmm, so I will remove the linter on this file